atlite/cutout.py

```diff
@@ -35,38 +35,68 @@
 import os, sys
 from warnings import warn
 from shapely.geometry import box
+from pathlib import Path
 
 import logging
 logger = logging.getLogger(__name__)
 
-from . import config
+from .config import ensure_config
 from . import datasets, utils
 
 from .convert import (convert_and_aggregate, heat_demand, hydro, temperature,
                       wind, pv, runoff, solar_thermal, soil_temperature)
+from .resource import Resources
 from .gis import GridCells
 from .data import requires_coords, requires_windowed, cutout_prepare
 
-class Cutout(object):
-    dataset_module = None
+def ensure_path(name, has_data, config):
+    """
+    Translate `name` argument to the full path of a cutout netcdf file or
+    directory of an old-style cutout
 
-    def __init__(self, name=None, data=None, cutout_dir=None, **cutoutparams):
+    In order, we look for:
+
+    - File with nc suffix in current working directory
```

Comment:
Why would we prioritise the current working directory, even if a `cutout_dir` is specified?
Shouldn't it be the other way round?